webpack.common.js

```diff
@@ -77,163 +78,176 @@ console.log('npm run watch  --themes_dir=themes/mytheme/client --css_file=myfile
+              quality: 60
             }
-          ]
-        },
-        // images
-        {
-          test: /\.(png|svg|jpe?g|gif)$/,
-          loader: 'url-loader',
-          options: {
-            limit: 4096, // in bytes
-            outputPath: IMG_DIR_CONFIG,
-            name: '[name].[ext]'
+          },
+          inline: {
+            limit: 1
           }
-        },
-        // fonts
-        {
-          test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
-          use: [
-            {
-              loader: 'file-loader',
-              options: {
-                outputPath: FONTS_DIR_CONFIG,
-                name: '[name].[ext]'
-              }
-            }
-          ]
-        },
-        // add jQuery
-        {
-          test: require.resolve('jquery'),
-          use: [
-            {
-              loader: 'expose-loader',
-              options: {
-                exposes: ['$', 'jQuery']
-              }
-            }
-          ]
         }
-        // {
-        //     test: /\.svg$/i,
-        //     use: 'svg-inline-loader'
-        // },
-      ]
+      }]
     },
+    {
+      test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
+      use: [{
+        loader: 'file-loader',
+        options: {
+          outputPath: FONTS_DIR_CONFIG,
+          name: '[name].[ext]'
+        }
+      }]
+    }
+    ]
+  },
 
-    // extra settings
-    resolve: {
+  // extra settings
+  resolve: {
 
-      // node modules to include
-      modules: [
-        path.join(__dirname, 'node_modules'),
-        path.resolve(NODE_DIR)
-      ],
+    // defines root folders for compatibility
+    roots: [
+      path.resolve('./../../'),
+      path.resolve('./../../public')
+    ],
 
-      // aliases
-      alias: {
-        site: path.resolve('./../../'),
-        PROJECT_ROOT_DIR: path.resolve('./../../')
-      }
-      // extensions: [".js", ".jsx"]
+    // //node modules to include
+    modules: [
+      path.join(__dirname, 'node_modules'),
+      path.resolve(NODE_DIR)
+    ],
+
+    // aliases
+    alias: {
+      'window.jQuery': require.resolve('jquery'),
+      $: require.resolve('jquery'),
+      jQuery: require.resolve('jquery'),
+
+      // react: require.resolve('react'),
+      // 'react-dom': require.resolve('react-dom'),
+
+      site: path.resolve('./../../'),
+      PROJECT_ROOT_DIR: path.resolve('./../../')
     },
 
-    plugins: [
-      // clean dist folder? Do not use as this will also delete all the images, etc...
-      // new CleanWebpackPlugin(
-      //     [path.resolve(variables.absolutePath, variables.distributionFolder)],
-      //     {
-      //           root: path.resolve(variables.absolutePath),
-      //           verbose: true,
-      //           dry: false
-      //     }
-      // ),
-      new webpack.ProvidePlugin(
-        {
-          $: 'jquery',
-          jQuery: 'jquery',
-          'window.jQuery': 'jquery'
-        }
-      )
-    ]
-  },
-  customConfig
+    fallback: {
```

Comment:
In case assets URL wasn't resolved just ignore it as it was with the previous version of webpack